scripts/install.sh

```diff
@@ -30,3 +30,13 @@ tee /etc/apt/sources.list.d/docker.list > /dev/null
 VERSION_STRING=$1
 apt-get -qq update
 apt-get -qq install docker-ce="$VERSION_STRING" docker-ce-cli="$VERSION_STRING" containerd.io docker-buildx-plugin docker-compose-plugin
+
+#create/enable swapping
+
+sudo fallocate -l 4G /swapfile
+sudo chmod 600 /swapfile
+sudo mkswap /swapfile
+sudo swapon /swapfile
+sudo swapon --show
+# swap partition to automatically activate after reboot
+echo /swapfile none swap sw 0 0 | sudo tee -a /etc/fstab
```

Comment:
Only run this if not added already. Start with `! grep swap /etc/fstab && ...`

Reply:
okay, I have done similar logic for running prune script in crontab. Please have a look
`! grep -q '0 0 * * 0 /concourse/scripts/prune_workers.sh' &&` line 164 in `concourse.tf`